lib/l10n_utils/fluent.py

```diff
@@ -84,6 +84,11 @@ def active_locales(self):
         # first resource is the one to check for activation
         return get_active_locales(self.resource_ids[0])
 
+    @cached_property
+    def active_home_locales(self):
+        # use mozorg/home to check for activation
+        return get_active_locales("mozorg/home.ftl")
+
```

Comment:
So, umm, instead of butchering the getter above it (potentially affecting gazillion page hits), it felt safer to just add a new one, doing just this one job checking for homepage languages separately, and using that in addition for the is_root condition. Maybe stupid, take this as more of a pseudo-code @stevejalim, and tear it apart as you see fit.

The hardcoded path string feels somewhat fragile here. Is there a better way?
(Also, some clever tests to cover the root specifics?)

Reply:
The path seems fine as long as we have a test that leverages this. That way if it gets refactored away at some later point in time a test will break to know we depended on that.

Reply:
I was thinking whether having an entry in SETTINGS would be more resilient, to keep the value of "homepage" locale file managed and visible in an obvious place, instead of having a string hardcoded somewhere deep in the code.

Reply:
I'm still not entirely sure the separate method is needed, as from I can infer it looks like the

https://github.com/mozilla/bedrock/blob/bc2d7909b1b401d166f42b7fa3fed01ba47cb557/lib/l10n_utils/fluent.py#L83-L85

right above the addition only does that, can't seem to find any use besides listing active locales (presumably for home page), but using the first ftl file which is the 404 file — and can't think of use case that would benefit from reading the activation threshold for a locale based on that 404 file per se… so I'm doubting whether a separate method is not overly safe, and the original method shouldn't be updated to look for the metadata in a homepage ftl file, instead of the 404 resource_id. (But wasn't able to inspect all the layers this gets used throughout the app, so can't tell there's not a specific use case that I wasn't able to find…)